app/src/lib/stores/pull-request-coordinator.ts

```diff
@@ -53,10 +51,18 @@ export class PullRequestCoordinator {
   ) {
     // register an update handler for the repositories store
     this.repositoriesStore.onDidUpdate(allRepositories => {
-      this.repositories = allRepositories.filter(
-        isRepositoryWithGitHubRepository
+      this.repositories = Promise.resolve(
+        allRepositories.filter(isRepositoryWithGitHubRepository)
       )
     })
+
+    this.repositories = this.repositoriesStore
```

Comment:
Is it worth adding a comment to indicate this triggers the initial load of repositories? 🤔